docs/60_Usage_Best_Practices.md

```diff
@@ -0,0 +1,302 @@
+            "name": "creatorId",
+            "description": "creatorId",
+            "type": "string",
+            "required": true,
+            "nullable": false
+        },
+        {
+            "name": "createdAt",
+            "description": "createdAt",
+            "type": "Date",
+            "required": true,
+            "nullable": false
+        },
+        {
+            "name": "updaterId",
+            "description": "updaterId",
+            "type": "string",
+            "required": true,
+            "nullable": false
+        },
+        {
+            "name": "updatedAt",
+            "description": "updatedAt",
+            "type": "Date",
+            "required": true,
+            "nullable": false
+        },
+        {
+            "name": "__STATE__",
+            "description": "__STATE__",
+            "type": "string",
+            "required": true,
+            "nullable": false
+        },
+        {
+            "name": "my_field",
+            "type": "string",
+            "required": true,
+            "nullable": false
+        }
+    ],
+    "indexes": [
+        {
+            "name": "_id",
+            "type": "normal",
+            "unique": true,
+            "fields": [
+                {
+                    "name": "_id",
+                    "order": 1
+                }
+            ]
+        },
+        {
+            "name": "createdAt",
+            "type": "normal",
+            "unique": false,
+            "fields": [
+                {
+                    "name": "createdAt",
+                    "order": -1
+                }
+            ]
+        },
+        {
+            "name": "my_field_index",
+            "type": "normal",
+            "unique": true,
+            "fields": [
+                {
+                    "name": "my_field",
+                    "order": 1
+                }
+            ]
+        }
+    ]
+}
+```
+
+In addition to the fields required by the CRUD Service (such as `__STATE__`, `updatedAt`, etc.), there is a mandatory string field called `my_field`, which is associated with a unique index named `my_field_index`.
+
+This indicates that the `my_field` field must be present in each document within the `my_single_view` collection, and it must have a unique value across all documents in the collection. The unique index ensures the uniqueness constraint on the `my_field` field, preventing duplicate values from being inserted into the collection.
+
+## Pagination
+
+A possible implementation to read data from the collection is to utilize client-side pagination by invoking the GET method with the query parameters `_sk` (skip) and `_l` (limit):
+
+The `_sk` parameter is used to specify the number of records to skip before starting to retrieve the result set. It allows you to navigate through different pages of data;
+
+The `_l` parameter represents the maximum number of records to retrieve in a single page. It determines the size of each page of data.
```

Comment:
Looking at the punctuation, I'll try to create a more consistent paragraph, with a bullet point to explain `_st` and `_l`.
Something like the following:

```suggestion
A possible implementation to read data from the collection is to utilize client-side pagination by invoking the GET method with the query parameters `_sk` (skip) and `_l` (limit):
- the `_sk` parameter is used to specify the number of records to skip before starting to retrieve the result set. It allows you to navigate through different pages of data.
- the `_l` parameter represents the maximum number of records to retrieve in a single page. It determines the size of each page of data.
```

Probably it's just me that I like bullet points, but I see the suggestion above somewhat clean. But I'll let you decide.

Reply:
I just missed the bullet point here :)